store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java

```diff
@@ -848,8 +848,8 @@ public GetMessageResult getMessage(final String group, final String topic, final
                                 selectResult.release();
                                 continue;
                             }
-
                             this.storeStatsService.getGetMessageTransferredMsgCount().add(cqUnit.getBatchNum());
+                            this.brokerStatsManager.incBrokerGetNumsWithoutSystemTopic(topic, cqUnit.getBatchNum());
```

Comment:
Using brokerStatsManager in the store module looks weird... It's better to add this counter in BrokerStatsManager#incBrokerGetNums.

Reply:
done